simpeg_drivers/components/factories/survey_factory.py

```diff
@@ -338,15 +349,22 @@ def _fem_arguments(self, data=None):
                 tx.rx_ids = np.r_[rx_id]
                 sources.append(tx)
 
+            source_ids = (
+                np.kron(
```

Comment:
There should be an equivalent np.tile or np.repeat operation that is more readable.